source/arroost/entities/arrows/time.js

```diff
@@ -62,7 +62,15 @@ export class ArrowOfTime extends Entity {
 		this.flaps.dom.style.fill.set(WHITE.toString())
 		this.flaps.dom.transform.scale.set([2 / 3, 2 / 3])
 
-		this.use(() => {
+		this.reusePosition()
+	}
+
+	reusePosition() {
+		if (this.positioner !== undefined) {
+			this.positioner.dispose()
+		}
+
+		this.positioner = this.use(() => {
```

Comment:
Moved this to a method so we can make wires change their dependencies to the new source/target cell